_topic_maps/_topic_map.yml

```diff
@@ -662,8 +662,14 @@ Topics:
     File: migrating-to-multi-payload
   - Name: Updating hosted control planes
     File: updating-hosted-control-planes
-# - Name: Troubleshooting an update
-#  File: updating-troubleshooting
+- Name: Troubleshooting a cluster update
+  Dir: troubleshooting_updates
+  Distros: openshift-enterprise, openshift-origin
```

Comment:
Might be fine w/ the space, but just in case:
```suggestion
  Distros: openshift-enterprise,openshift-origin
```

Reply:
removing space!